### PR TITLE
use FQDN for in-cluster services in test cluster

### PR DESCRIPTION
**Pull request by BenTheElder**
we should also consider doing this for our prow deployment

/cc @MrHohn 

Holding so I can bump the images to pick up the kubetest changes
/hold

### CODE REVIEW

File: boskos/janitor/deployment.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ spec:
       - name: boskos-janitor
         image: gcr.io/k8s-testimages/janitor:v20180725-c421ed4f9
         args:
+        - --boskos-url=http://boskos.test-pods.svc.cluster.local
```
Comment (MrHohn) [inline boskos/janitor/deployment.yaml]: Here and below, we should include the trailing dot for FQDN:
```
http://boskos.test-pods.svc.cluster.local.
```

Comment (BenTheElder) [reply boskos/janitor/deployment.yaml]: d'oh, done!

**Comment by BenTheElder** [comment]
added an image bump

**Comment by BenTheElder** [comment]
test run here: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-canary/22152/

**Comment by BenTheElder** [comment]
/hold cancel
test passed

**Pull request merged**